pkg/plugins/plugins.go

```diff
@@ -109,27 +110,45 @@ func (p *Plugin) activateWithLock() error {
 }
 
 func load(name string) (*Plugin, error) {
-	storage.Lock()
+	return loadWithRetry(name, true)
```

Comment:
@cpuguy83, no it doesn't have to be! However, I modeled it after *pkg/plugins/client.go* where separate functions are used for the equivalent case. I think we should be consistent either way.

*pkg/plugins/client.go*
````
  // Call calls the specified method with the specified arguments for the plugin.
  // It will retry for 30 seconds if a failure occurs when calling.
  func (c *Client) Call(serviceMethod string, args interface{}, ret interface{}) error {
      return c.callWithRetry(serviceMethod, args, ret, true)
  }
  
  func (c *Client) callWithRetry(serviceMethod string, args interface{}, ret interface{}, retry bool) error {
      var buf bytes.Buffer
      if err := json.NewEncoder(&buf).Encode(args); err != nil {
          return err
      }
````